### PR TITLE
Update UI bundle to pick up changes for Scala

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

**Comment by jamiltz** [comment]
@RichardSmedley it looks like some links are broken on the preview https://deploy-preview-61--cb-docs-staging.netlify.com/home/index.html

And the nav link https://deploy-preview-61--cb-docs-staging.netlify.com/scala-sdk/1.0alpha/start-using-sdk.html

I tried to find the root cause but can't find it.

**Comment by RichardSmedley** [comment]
Needs to be https://deploy-preview-61--cb-docs-staging.netlify.com/scala-sdk/1.0alpha/hello-world/start-using-sdk.html
for this one.
If built from more recent updates in GitHub, 1.0 replaces 1.0alpha

**Comment by jamiltz** [comment]
Ok! Preview build in progress.

**Comment by RichardSmedley** [comment]
Thanks, James - netify has the docs up fine at:
https://deploy-preview-61--cb-docs-staging.netlify.com/scala-sdk/1.0/hello-world/start-using-sdk.html
https://deploy-preview-61--cb-docs-staging.netlify.com/go-sdk/2.0/hello-world/start-using-sdk.html

**Comment by RichardSmedley** [comment]
https://deploy-preview-61--cb-docs-staging.netlify.com/java-sdk/3.0/hello-world/start-using-sdk.html
https://deploy-preview-61--cb-docs-staging.netlify.com/dotnet-sdk/3.0/hello-world/start-using-sdk.html
.
but Scala is still not in the dropdown nav

**Comment by jamiltz** [comment]
@RichardSmedley to appear on the SDK related pages dropdown, `scala-sdk/1.0` must be added to this for-loop as well https://github.com/couchbase/docs-ui/blob/master/src/partials/header-content.hbs#L70

I've also noticed that for other SDKs, `start-using-sdk.adoc` is located in the `ROOT` folder

https://deploy-preview-61--cb-docs-staging.netlify.com/c-sdk/2.10/start-using-sdk.html
https://deploy-preview-61--cb-docs-staging.netlify.com/dotnet-sdk/2.7/start-using-sdk.html
https://deploy-preview-61--cb-docs-staging.netlify.com/go-sdk/1.5/start-using-sdk.html

From I understand, it should be the same for sdk-docs-scala in order for the SDK related pages dropdown to work as expected.

**Comment by RichardSmedley** [comment]
Ah, nice find James. :+1: 

Yes, they've all moved for the new Information Architecture in SDK '3.0'
howtos, concepts, refs, and project docs all have separate modules too

Maybe there's a workaround?
Shall I put Scala 3.0 in L70 for now?

**Comment by amarantha-k** [comment]
Richard, I don't think there is a quick and easy workaround, looks like this requires the underlying code to be updated. Let's review the modules too and figure out what our next steps should be.

**Comment by jamiltz** [comment]
Yes, I would also move all [`.adoc` files](https://github.com/couchbase/docs-sdk-scala/tree/release/1.0alpha/modules) in the `ROOT` module similarly to other SDK repos https://github.com/couchbase/docs-sdk-java/tree/release/2.7/modules, that way the Scala SDK pre-release isn't blocked by a docs-ui issue..supporting non-ROOT modules for SDK components doesn't look like a trivial change.

**Comment by RichardSmedley** [comment]
As discussed in another channel, non-ROOT modules used for best practice and will not be moving.
It has been suggested that :page-aliases: may work with paths, to facilitate switching - eg
../../ROOT/thisfile.adoc as a page-alias

File: staging-antora-playbook.yml [hunk 1]
```diff
@@ -40,6 +40,7 @@ content:
   - url: https://github.com/couchbase/docs-sdk-php.git
   - url: https://github.com/couchbase/docs-sdk-python.git
   - url: https://github.com/couchbase/docs-sdk-scala.git
+    branches: [release/1.0alpha]
```
[the remark this answers is not in the document]
Comment (jamiltz) [reply staging-antora-playbook.yml]: Yes, let's try.

```suggestion
```

**Comment by jamiltz** [comment]
@RichardSmedley Thanks for taking the time to work through the road blocks!

**Pull request merged**